mitiq/calibration/settings.py

```diff
@@ -186,10 +187,14 @@ def __init__(
         self.circuit_types = circuit_types
         self.num_qubits = num_qubits
         self.circuit_depth = circuit_depth
+        self.circuit_seed = circuit_seed
 
-    def make_circuits(self) -> List[BenchmarkProblem]:
+    def make_circuits(
+        self, seed: Optional[int] = None
+    ) -> List[BenchmarkProblem]:
         """Generate the circuits to run for the calibration experiment.
-
+        Args:
+            seed(optional): a random seed for (mirror) circuit generation.
```

Comment:
```suggestion
            seed(optional): A random seed for (mirror) circuit generation.
```

Reply:
Moved docstring for `seed` to `Settings` since it is no longer an explicit argument of `make_circuits`.